ROADMAP.md

```diff
@@ -298,14 +300,14 @@ Status | Command | Description
 
 Status | Command | Description
 ---|--------|------------------------------
-:warning:   | "{char}	        | use register {char} for the next delete, yank, or put
+:warning: :red_circle:  | "{char}	        | use register {char} for the next delete, yank, or put
```

Comment:
Thanks for your info, I'll mark it as `read only registers not supported yet` first :) Then you may want to give it a better description if you like.